scripts/biomedical/human1 data/scripts/format_genes_geneRoles.py

```diff
@@ -0,0 +1,34 @@
+def main():
+    import sys
+    import os
+    import numpy as np
+    import pandas as pd

```

Comment:
imports are defined globally outside the main function

Reply:
Fixed